### DISCUSSION TITLE
Poster Features

### DISCUSSION THREAD

**Discussion by petersem** [petersem/posterr#5] [Ideas]
Right now, the plan for Poster is that it generates cards for:

- Now showing titles (plex only) - with a progress bar indicating watched progress and if direct or transcoding.
- Random on-demand titles from specified libraries (plex only) 
- Coming soon titles from Sonarr (with the option to only show season premieres) 
- Coming soon titles from Radarr
- Docker integration

Additional Features that I am planning are:
- ~~Now playing for songs playing, showing album cover art and song details.~~
- An optional 'health' card that will show the availability of Plex, Sonarr, Radarr, Bazarr, Sab, etc.  Just a simple green/red dashboard. Maybe something where you can build rules to call api's or urls and check for values/text in the results.
- Genre search for on-demand cards
- The ability to target a specific IP, player device, devices, or all devices. This way, you could spool up a version specifically for a user. 
- A custom image card, where you can supply fixed images to display
-  A custom text card
- A card randomise function which will mix the order of the cards. (currently now showing, on-demand, coming soon tv, coming soon movies)
- A custom URL page that would load the contents of a web page into a card.
- A selection of different slide transition effects. (currently only has a slide and a fade)
- Option to have an installer for Windows, Linux and Mac.  (there's a lot of users that don't have Docker in their repertoire) 
- I'm still not sure if i should bother with Kodi, Emby, but still planning Jellyfin integration.
- Movie trailers
- ~~Setup page with password protection to set all values for the app. Including optional 'dark' mode.~~

Thoughts/suggestions?

**Comment by savaloi** [comment]
I like all the ideas! I look forward to seeing them! I wonder if the health section might be more easily covered by integration with something like Monitorr etc, instead of re-inventing the wheel?

**Reply by petersem** [reply]
I thought about this. Jury is still out 😆

**Reply by petersem** [reply]
@savaloi Have you tried the latest version out? It has come a long way in the last 17 days.

**Comment by binarygeek119** [comment]
please dont drop planned support for emby

**Reply by petersem** [reply]
Jellyfin is next, and I am hopeful this will be very similar to emby. But i havent reviewed this in any details as yet.

**Comment by ZeFish** [comment]
Would be nice to be able to get only the last 10 added content for "On Demand". Something in the line of "Just released"

**Reply by petersem** [reply]
This is available now

**Comment by GeekDoc** [comment]
Is this project still active?  I'd really like to see it back on the Unraid Community Apps.

**Reply by petersem** [reply]
Currently waiting for it to be approved in the community apps store. Hopefully sometime in the next day or so.

**Reply by GeekDoc** [reply]
Great news!  I'll keep checking.

**Reply by petersem** [reply]
It's there now

**Reply by GeekDoc** [reply]
Installed and working well!

**Reply by petersem** [reply]
Nice one 👍
